src/wayland_egl/pxWindowNative.cpp

```diff
@@ -582,7 +582,7 @@ void pxWindowNative::runEventLoop()
       }
     }
 
-    rtLogInfo("pxcore framerate: %d", framerate);
+    rtLogDebug("pxcore framerate: %d", framerate);

```

Comment:
Please leave this logging statement as info.  It's important for it to be logged by default at the info level, and it's only logged once at startup.